youtube_dl/extractor/deezer.py

```diff
@@ -89,3 +86,62 @@ def _real_extract(self, url):
             'thumbnail': playlist_thumbnail,
             'entries': entries,
         }
+
+
+class DeezerAlbumIE(DeezerBaseInfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?deezer\.com/(../)?album/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://www.deezer.com/fr/album/67505622',
+        'info_dict': {
+            'id': '67505622',
+            'title': 'Last Week',
+            'uploader': 'Home Brew',
+            'thumbnail': r're:^https?://(e-)?cdns-images\.dzcdn\.net/images/cover/.*\.jpg$',
+        },
+        'playlist_count': 7,
+    }
+
+    def _real_extract(self, url):
+        album_id, webpage, data = self.get_data(url)
+
+        album_title = data.get('DATA').get('ALB_TITLE')
+        album_uploader = data.get('DATA').get('ART_NAME')
+        album_thumbnail = self._search_regex(
+            r'<img id="naboo_album_image".*?src="([^"]+)"', webpage,
+            'album thumbnail')
+
+        entries = []
+        for s in data.get('SONGS').get('data'):
+            formats = [{
+                'format_id': 'preview',
+                'url': s.get('MEDIA')[0].get('HREF'),
```

Comment:
Same.

Reply:
I've done `s.get('MEDIA', [{}])[0].get('HREF')` so that if there is no key `MEDIA`, there is no error on `[0]`.